code/lib/cli/src/generators/baseGenerator.ts

```diff
@@ -246,6 +246,19 @@ export async function baseGenerator(
     ...extraAddonsToInstall,
   ].filter(Boolean);
 
+  const shouldUseSWCCompiler =
+    builder === CoreBuilder.Webpack5 &&
+    (projectType === ProjectType.HTML ||
```

Comment:
perhaps `projectType === ProjectType.SERVER` should be here too

Reply:
Good catch!